server/service/src/sync/translations/invoice.rs

```diff
@@ -575,12 +569,10 @@ impl SyncTranslation for InvoiceTranslation {
             linked_transaction_id: linked_invoice_id,
             entry_date: created_datetime.date(),
             entry_time: to_legacy_time(created_datetime),
-            ship_date: shipped_datetime
-                .map(|shipped_datetime| date_from_date_time(&shipped_datetime)),
-            arrival_date_actual: delivered_datetime
-                .map(|delivered_datetime| date_from_date_time(&delivered_datetime)),
```

Comment:
I kind of think these were valid mappings... not sure though. Could you ask around the old guard a bit to see if they actually make sense?

ship_date being "shipped_datetime" kinda tracks. 

Also I haven't checked how these might play out to the transfer processors, any thoughts there?

Reply:
OG has it as an editable field on itself, right? 
<img width="687" height="117" alt="Screenshot 2025-10-02 at 1 24 58 PM" src="https://github.com/user-attachments/assets/edd96f92-02e2-4cd3-9c6e-006963f6e3cf" />

Idk can add it back, but what if the user is entering the data on a later day? Then it'll show the wrong dates? 👀